docs/mainnet.md

```diff
@@ -0,0 +1,174 @@
+  is fine.
+* `--monerod-host HOSTNAME_OR_IP` and `--monerod-port PORT_NUM`: Ideally, you have your
+  own node on the local network and will use these values. If that is not an
+  option, our default uses `node.sethforprivacy.com`.
+* `--libp2p-port PORT`. The default is `9900`. Use this flag when creating multiple
+  swapd instances on the same host.
+* `--rpc-port PORT`. The default is `5000`. Use this flag when creating multiple
+  swapd instances on the same host.
+* `--log-level LEVEL`. If you want to see debug logs, you can set `LEVEL` to `debug`. If you want less logs, you can set it to `warn` or `error`.
+
+> Note: please also see the [RPC documentation](./rpc.md) for complete documentation on available RPC calls and their parameters.
+
+## Relayer
+ 
+The Ethereum network requires that users have ether in an account to be able to execute any transactions from that account. For ETH-takers, this means that they would need to have an already-funded account to claim their swap funds. However, this is not ideal as it harms the private of the swap users. A workaround is to have users relay transactions on behalf of others, meaning that the relayer would pay the gas fee for the swap claim transaction and receive a small portion of the funds in return.
+
+To run a node as a relayer, pass the `--relayer` flag to `swapd`:
+```bash
+./bin/swapd --env stagenet --eth-endpoint MAINNET_ENDPOINT --relayer
+```
+
+Note: the current fee sent to relayers is 0.009 ETH per swap. Subtract the gas cost from this to determine how much profit will be made. The gas required to do a relayer-claim transaction is `102048` gas. Multiply this by the transaction gas price for the gas cost. The gas price is set via oracle unless you manually set it with the `personal_setGasPrice` RPC call.
+
+## swapcli commands
+
+`swapcli` is used to interact with `swapd`, ie. for finding peers and offers on the network and making/taking swaps.
+
+Some useful commands are:
+* `swapcli balances`: check your ETH and XMR addresses and balances.
+* `swapcli ongoing`: check the status of all ongoing swaps.
+* `swapcli past`: see all your past swaps.
+* `swapcli get-offers`: see all your currently advertised offers.
+
+You can see all available commands with `swapcli -h`.
+
+## Monero Taker 
+
+1. Check your Ethereum address and balance and ensure your address is funded:
+```bash
+./bin/swapcli balances
+```
+
+2. Search for existing XMR offers using `swapcli`:
+```bash
+./bin/swapcli discover --provides XMR --search-time 3 --swapd-port 5001
+# [[/ip4/127.0.0.1/udp/9934/quic-v1/p2p/12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv /ip4/127.0.0.1/udp/9934/quic-v1/p2p/12D3KooWC547RfLcveQi1vBxACjnT6Uv15V11ortDTuxRWuhubGv]]
+```
+
+3. Query a returned peer as to how much XMR they can provide and their preferred exchange rate (replace `"--multiaddr"` field with one of the addresses returned in the above step):
```

Comment:
Should we be mentioning ERC20 tokens at all?

Reply:
yes, will add a section to maker/taker about them